tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -726,7 +737,7 @@ pub fn verify<G: Curve>(
     }
 
     let mut t = vec![G::Zp::ZERO; n];
-    G::Zp::hash(
+    G::Zp::hash_128bit(
```

Comment:
is it from the original ZK formulation ?

Reply:
this is from Benoit's remark about optimizing the G2 multi exp

Reply:
ok !